acceptance-tests/data_source_mongodbatlas_accesslist_api_key_test.go

```diff
@@ -1,4 +1,4 @@
-package mongodbatlas
+package acceptancetests
```

Comment:
why not `test` https://github.com/golang-standards/project-layout/tree/master#test